.github/workflows/build.yml

```diff
@@ -39,7 +40,9 @@ jobs:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           submodules: recursive
-      - uses: dtolnay/rust-toolchain@1.68.2
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
These should still be pinned to _something_

Reply:
The recommended way of doing this is described [here](https://github.com/dtolnay/rust-toolchain#inputs). The `@rev` is used to select the Rust toolchain (not the action revision). This change is needed to use a Rust version defined by a variable (See `RUST_VERSION` above).

Reply:
No, we should be pinning both the action and the Rust toolchain separately. Something like `uses: dtolnay/rust-toolchain@0e66bd3e6b38ec0ad5312288c83e47c143e6b09e # v1`. The old pin was also wrong, since the Rust versions revs are floating branches and updated whenever master is (from the looks of it, anyway).

Reply:
I obviously agree that we should pin both version. I'm not quite sure if pinning the action will work tho (as the `@rev` is parsed as the toolchain version). I will test that.

Reply:
Pinned in 914e1a6